src/generateIncludes.js

```diff
@@ -96,5 +97,15 @@ export default function generateIncludes(simpleAST, type, root, options) {
     }
   });
 
+  if (isConnection(type)) {
+    let node = simpleAST.fields.edges.fields.node;
+    let fields = [];
+    _.forIn(node.fields, (field, key) => {
+      if (!field.fields.hasOwnProperty('edges')) {
+        fields.push(key);
+      }
+    });
+    result.attributes = result.attributes.concat(fields);
+  }
```

Comment:
Just FYI this code was removed with https://github.com/mickhansen/graphql-sequelize/commit/26c1955b8a30f78cb7cab827735b1420fdc71899

We now support proper nested includes with nested connections.